PeakLocalisations/Localisation/Features/WidgetSetup.xcstrings

```diff
@@ -178,6 +268,12 @@
       "comment" : "Separate instructions with a single new line",
       "extractionState" : "manual",
       "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Hinweis: Der StandBy Modus funktioniert nur auf einem iPhone mit iOS 17 oder neuer.\nGehe in die Einstellungen > StandBy und stelle sicher, dass StandBy aktiviert ist.\nVerbinde dein iPhone mit einem Ladegerät und kippe es in eine horizontale Position.\nDrücke den Seitenschalter.\nStreiche links oder rechts, um zwischen den verschiedenen Ansichten zu wechseln. Streiche hoch oder runter um zwischen den verschiedenen Optionen der Ansichten zu wechseln."
```

Comment:
I actually changed it to just "StandBy" (as Apple does). Search for "StandBy" on the [official Apple iOS 17 page](https://www.apple.com/de/ios/ios-17/) and click on the plus in the lower right of the picture to see a text.

Similar for the reference to the minimum requirements btw. IMHO it's more natural to reference a single device than multiple ones, since most people only have one (active) iPhone.